includes/class-cache.php

```
@@ -618,6 +621,24 @@ public function get_cache_file_url( $type = 'html' ): string {
+		 *
+		 * @since 1.9.0
+		 */
+		private function maybe_apply_used_css( string $buffer ): string {
+			$file_opts = $this->options['file_optimisation'] ?? array();
+			if ( empty( $file_opts['removeUnusedCSS'] ) ) {
+				return $buffer;
+			}
+
+			$used_css = new Used_CSS( $this->options );
+			return $used_css->process_buffer( $buffer );
+		}
```

PHPMD (2.15.0)

[error] The method maybe_apply_used_css is not named in camelCase. (undefined)

(CamelCaseMethodName)